patches/chrome-browser-ui-views-page_info-page_info_cookies_content_view.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc b/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
+index 9c37ceebe38cee1f2acb7ffd49b1364647ed80e9..2ce1791ad7279267823ccc7c2cf2cfd9fe8894ee 100644
+--- a/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
++++ b/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
+@@ -176,6 +176,7 @@ void PageInfoCookiesContentView::SetBlockingThirdPartyCookiesInfo(
+       NOTREACHED_NORETURN();
+   }
+ 
++  BRAVE_HIDE_COOKIES_BLOCK_CONTROL
```

Comment:
can't we just disable `kPageInfoCookiesSubpage`?

Reply:
Updated to disable `kPageInfoCookiesSubpage`.